core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -187,4 +187,48 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareT
     }
     checkReplicaDistribution(assignment, rackInfo, rackInfo.size, alteredNumPartitions, replicationFactor)
   }
+
+  @Test
+  def testDescribeAndListTopicsMarkedForDeletion() {
+    val brokers = List(0)
+    val topic = "testtopic"
+    val markedForDeletionDescribe = "MarkedForDeletion"
+    val markedForDeletionList = "marked for deletion"
+    TestUtils.createBrokersInZk(zkUtils, brokers)
```

Comment:
We have a lot of code that tests that checking the relevant zk path means that a topic is marked for deletion. Here, we just need to make sure that if the relevant zk method returns true/false, the output is fine.

Sadly, we don't have the method we want, but it's trivial to add a `ZkUtils.isTopicMarkedForDeletion` with an implementation that is simply: `zkUtils.pathExists(getDeleteTopicPath(topic))`.